src/pages/DataLoader/DataLoader.tsx

```diff
@@ -0,0 +1,321 @@
+import React, { useRef } from 'react';
+
+import './styles.scss';
+
+import link from '@/assets/icons/link.svg';
+import { Button } from '@/components/Button/Button';
+import { useDataLoader } from '@/hooks/useDataLoader';
+import { useWorkspace } from '@/hooks/useWorkspace';
+
+import AccessPolicyDescription from './descriptions/AccessPolicyDescription';
+import STACDescription from './descriptions/STACDescription';
+
+const DataLoader = () => {
+  const { activeWorkspace } = useWorkspace();
+  const {
+    file,
+    setFile,
+    fileName,
+    setFileName,
+    state,
+    setState,
+    message,
+    setMessage,
+    running,
+    setRunning,
+    validationErrors,
+    setValidationErrors,
+    fileType,
+    setFileType,
+  } = useDataLoader();
+
+  const fileInputRef = useRef<HTMLInputElement>(null);
+
+  const renderHeader = () => {
+    return (
+      <div className="header">
+        <div className="header-left">
+          <h2>Data Loader</h2>
+        </div>
+        <div className="header-right">
+          <img alt="Members" src={link} />
+          <div className="header-right-text">
+            <span className="header-right-title">Data Loader</span> allows you to validate, upload
+            and harvest STAC files directly into your workspace.
```

Comment:
Needs to be clearer that an item, collection or catalog can be uploaded.

Also can you upload an item that has a STAC collection ID, does that mean the STAC collection gets generated? What happens to the existing `links` inside of item/collection/catalog?

It may be more of a question to @hcollingwood, but we will still need to explain it on the frontend for the user